packages/@storefront/navigation/src/filter-refinement-controls/index.ts

```diff
@@ -7,11 +8,24 @@ class FilterRefinementControls extends ValueRefinementControls {
     return 'filterControls';
   }
 
+  get selectMultipleRefinements(): string {
+    switch (this.props.storeSection) {
+      case StoreSections.PAST_PURCHASES:
+        return 'selectMultiplePastPurchaseRefinements';
+      case StoreSections.SEARCH:
+        return 'selectMultipleRefinements';
+    }
+  }
+
   fetchMoreRefinements = () => {
     if (this.state.more) {
       this.state.moreRefinements();
     }
   }
+
+  selectMatchedRefinements = (event: Tag.Event, refinements: RefinementControls.SelectedRefinement[]) => {
+    this.actions[this.selectMultipleRefinements](this.props.navigation.field, refinements.map((refinement) => refinement.index));
+  }
 }
 
```

Comment:
Add props definition for `enableSelectAll`, follow `rangeRefinementControls` for an example on how to do that